Wikipedia/Code/WMFFeedImage.m

```diff
@@ -10,6 +11,7 @@ @implementation WMFFeedImage
 + (NSDictionary *)JSONKeyPathsByPropertyKey {
     return @{WMF_SAFE_KEYPATH(WMFFeedImage.new, canonicalPageTitle): @"title",
              WMF_SAFE_KEYPATH(WMFFeedImage.new, imageDescription): @"description.text",
+             WMF_SAFE_KEYPATH(WMFFeedImage.new, imageDescriptionIsRTL): @"description.lang",
              WMF_SAFE_KEYPATH(WMFFeedImage.new, imageURL): @"image.source",
```

Comment:
I think you have to bump the model version by adding
```+ (NSUInteger)modelVersion {
    return 2;
}``` 
when you update the model. We should also verify that this continues to work with old cached data after this change.